test/support/factory.ex

```diff
@@ -1,12 +1,11 @@
 defmodule GRPC.Factory do
   @moduledoc false
 
-  alias GRPC.Channel
-  alias GRPC.Credential
+  use GRPC.Factories.Channel
+  use GRPC.Factories.Client.Stream
 
-  @cert_path Path.expand("./tls/server1.pem", :code.priv_dir(:grpc))
-  @key_path Path.expand("./tls/server1.key", :code.priv_dir(:grpc))
-  @ca_path Path.expand("./tls/ca.pem", :code.priv_dir(:grpc))
+  # Protobuf factories
+  use GRPC.Factories.Proto.HelloWorld
```

Comment:
I'd keep things simple and keep all factories here for now. I know it seems cleaner to split up modules, but the fact that we have to encapsulate everything in macros smells bad for me.

There aren't too many factories yet to warrant this split IMO

Reply:
Yeah! That makes sense. I started using this and I thought I would be adding more stuff, but it's not the case. I'll make that change